atomicapp/utils.py

```diff
@@ -214,3 +215,9 @@ def getRoot():
             return HOST_DIR
         else:
             return "/"
+
+    # generates a unique 12 character UUID
+    @staticmethod
+    def getUniqueUUID():
```

Comment:
When this patch gets merged, please replace my `random` stuff with this method (https://github.com/projectatomic/atomicapp/blob/master/atomicapp/install.py#L90)